apps/src/applab/levels.js

```diff
@@ -24,8 +25,7 @@ levels.simple = {
         <value name="ID"><block type="text"><title name="TEXT">id</title></block></value> \
         <value name="HTML"><block type="text"><title name="TEXT">html</title></block></value></block>'
   ),
-  startBlocks:
-    '<block type="when_run" deletable="false" x="20" y="20"></block>',
+  startBlocks: `<block type="${BLOCK_TYPES.whenRun}" deletable="false" x="20" y="20"></block>`,
```

Comment:
I can't begin to guess why we'd be listing Blockly stuff in App Lab startBlocks...